test/backend/routes.spec.js

```diff
@@ -1616,10 +1612,7 @@ describe('Routes:', () => {
             }));
         });
 
-        // TODO - Getting intermittent FetchError: request to http://localhost:9494/queries failed, reason: socket hang up
         it("queries POST /queries fails when it can't connect to the plotly server", function() {
-            this.timeout(70 * 1000);
```

Comment:
This is bigger than default time out of 60s. Capping it to 60 _may_ cause more intermittent timeouts in future..

Mentioning this since we may need to re-introduce it if it is causing intermittent failures later..

Reply:
It's on purpose. If we see the timeouts, we'll increase the default timeout instead, so that all the specs benefit from the increase.